source/how-we-work/how-to-run-a-design-system-team-ceremony-or-workshop.html.md

```diff
@@ -0,0 +1,165 @@
+---
+title: How to run a Design System team ceremony or workshop
+weight: 5
+---
+
+# How to run a Design System team ceremony or workshop 
+
+
+This document details the principles, tools and resources we use when planning and running a ceremony or workshop on the Design System team.
+## Principles
+
+There are some principles that are important to remember when planning and facilitating a ceremony or workshop on the Design System team;
+
+- consider if there are asynchronous methods you can use to get the information you need. With remote working it is easy to experience video call fatigue.
```

Comment:
For an iteration, but might be worth adding some docs on async communication?

Reply:
Yep agree